posthog-react-native/src/hooks/useFeatureFlag.ts

```diff
@@ -18,3 +19,26 @@ export function useFeatureFlag(flag: string): string | boolean | undefined {
 
   return featureFlag
 }
+
+export function useFeatureFlagWithPayload(flag: string): [string | boolean | undefined, JsonType | undefined]   {
+  const posthog = usePostHog()
+
+  const [featureFlag, setFeatureFlag] = useState<boolean | string | undefined>(posthog?.getFeatureFlag(flag))
+  const [featureFlagPayload, setFeatureFlagPayload] = useState<JsonType | undefined>(posthog?.getFeatureFlag(flag))
```

Comment:
```suggestion
  const [featureFlagPayload, setFeatureFlagPayload] = useState<JsonType | undefined>(posthog?.getFeatureFlagPayload(flag))
```

Reply:
Made the change! I did not sleep enough when I made this haha